### DISCUSSION TITLE
Qual local que  aceita iniciantes, Empresa de grande, media ou pequeno porte ou Startups?

### DISCUSSION THREAD

**Discussion by AdemilsonSimiao** [frontendbr/forum#2253] [Discussão]
Estou pesquisando onde é melhor para quem está querendo entrar no mercado para ganhar experiência.
Empresas de grande, media ou pequeno porte, startups ou sites para freelances, ou tem outro caminho?

**Comment by MiqueiasGFernandes** [comment]
Honestamente, onde for possível. O mercado pra quem não tem experiência não está muito fácil.
Até mesmo com pequenos projetos pessoais desenvolvidos aqui mesmo no Github.

Mas geralmente as empresas mais acolhedoras tendem a ser as menores (Pelo que vejo)

**Comment by renatoknot** [comment]
Também gostaria de saber, está muito concorrido o mercado de JR. Ainda mais para Front-end que é onde a galera começa geralmente.

**Reply by AdemilsonSimiao** [reply]
Olá Renato! Para quem está fazendo graduação um bom caminho é os programas de estágio, concorrido, mas não exigem muito.
Dois bons lugares e a CIEE (ciee.org.br) o NUBE (nube.com.br), outros dois lugares pra quem quer se capacitar é a DIO (dio.me) e a RECODE(recode.org.br), esses dois últimos ajuda muito no seu conhecimento técnico e com ótimos projetos para seus portfólios.

**Reply by renatoknot** [reply]
Infelizmente não posso fazer estágio sou formado desde 2019... Mas infelizmente ainda não atuo, a Dio já me inscrevi em alguns bootcamps mas não gostei muito da didática, pode ser algo pessoal meu. Ultimamente tenho feito curso na Udemy
